components/browser/state/src/main/java/mozilla/components/browser/state/state/TabSessionState.kt

```diff
@@ -11,20 +11,24 @@ import java.util.UUID
  *
  * @property id the ID of this tab and session.
  * @property content the [ContentState] of this tab.
+ * @property parentId the parent ID of this tab.
```

Comment:
nit: this reads a bit mechanical and doesn't really say what a parent id is. Maybe something like "The ID of the parent tab or null if this tab has no parent"?

Reply:
Ah yes. I've added more details here now i.e. what is a parent tab. Good catch. :)